### PR TITLE
fix(demos): codepen demos load svg assets

### PR DESCRIPTION
**Pull request by ThomasBurleson**
The CodePen demo launches now correctly load the external, svg-assets (and post-load $templateCache) used in some of the md-icon demos.

*  now defines external module  'material.svgAssetsCache'
*  now defines extra svgs for 'ic_euro_24px.svg', and 'ic_card_giftcard_24px.svg'

### CODE REVIEW

**Comment by ThomasBurleson** [comment]
@topherfangio - can you review ?

File: docs/app/js/codepen.js [hunk 1]
```diff
@@ -184,18 +185,18 @@
...
-    // Required to make codepen work. Demos define their own module when running on the
-    // docs site.  In order to ensure the codepen example can use the asset-cache, the
+    // Required to make codePen work. Demos define their own module when running on the
+    // docs site.  In order to ensure the codepen example can use the svg-asset-cache.js, the
     // module needs to match so that the $templateCache is populated with the necessary
     // assets.
+
     function replaceDemoModuleWithCodepenModule(file) {
       var matchAngularModule =  /\.module\(('[^']*'|"[^"]*")\s*,(\s*\[([^\]]*)\]\s*\))/ig;
+      var modules = "['ngMaterial', 'ngMessages', 'material.svgAssetsCache']";
```
Comment (DevVersion) [inline docs/app/js/codepen.js]: That's a very nice way to fix the `asset-cache`.

**Comment by DevVersion** [comment]
@ThomasBurleson - What do you think about a Gulp Task for Generating the Assets (certainly after your PR)? [See here for example](https://github.com/DevVersion/material/blob/fix/docs-assets-cache/docs/gulpfile.js#L139-L162)

**Comment by topherfangio** [comment]
@ThomasBurleson LGTM

**Comment by ThomasBurleson** [comment]
@DevVersion - Submit a PR for the Gulp Task to build the asset-cache. Notice that it is now `svg-assets-cache.js`.

File: docs/app/js/codepen.js (hunk 1 above)
Comment (ThomasBurleson) [inline docs/app/js/codepen.js]: Thx. It was a brain-teaser at first.

**Pull request closed without merging**